tests/APIv0/SmokeTest.php

```diff
@@ -61,6 +61,23 @@ public function setTestUser($testUser) {
         return $this;
     }
 
+    /**
+     * Get a single discussion from an array of Discussions.
+     *
+     * @param array $discussions
+     * @return array $discussion
```

Comment:
The format of the [@return tag](http://docs.phpdoc.org/references/phpdoc/tags/return.html) is:

```
@return [Type] [<description>]
```

Including the name of the variable you're returning isn't necessary, because the name doesn't come along as part of the result.